### ISSUE TITLE
getOoxml() steals focus away from modern comments

### ISSUE THREAD

**Issue by andreas-schoch** [OfficeDev/office-js#2309]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

# getOoxml() steals focus from modern comments
Using `getOoxml()` to get the ooxml of either the whole document body or for a single range, steals the focus away when user tries to insert a new comment using the context menu (e.g right click --> New Comment). 

Bug happens for both legacy edge and chromium based edge (webview2) on windows.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: 2111 (Build 14701.20226 Click-to-Run)
* Operating System: Windows 10 Pro (20H2)
* Browser (if using Office on the web): -
 
## Expected behavior
The comment draft textfield should **not** lose focus when getOoxml() is called.

## Current behavior
The comment draft textfield loses focus when getOoxml() is called. If user doesn't notice and continues to write, the text is added to the document instead.

_(Note that bug does not happen when inserting a comment via comments pane only when the draft is shown side-by-side with the document which seems to be the default when using the context menu)._

## Steps to reproduce
Make sure modern comments are enabled (see second link below)
1. Run the gist below in scriptlab.
2. Click either of the two buttons (to start an interval which calls getOoxml() every second)
3. Right click in the document and select "New Comment"
4. Try to write a comment in the textfield which will appear after step 3.

## Link to live example(s)
1. Scriptlab gist: https://gist.github.com/andreas-schoch/2c1de47a4f6c4e989243c05d52c87898

# Provide additional details
1. About modern comments: https://techcommunity.microsoft.com/t5/microsoft-365-blog/introducing-modern-comments-in-microsoft-word/ba-p/2263182

**Comment by greysteil** [comment]
@wangyun-microsoft @gergzk would you consider reopening this? I've confirmed that the bug is still present in Word version 2404 (Build 17531.20152).

Fetching OOXML is a core part of our add-in, and I imagine a lot of Word add-ins use it since it provides so much flexibility. We use it to get the current text with a few tweaks (e.g., accounting for tables and placeholder text), as well as formatting information.

We've had a couple of our end users complain to us about the result of this bug. The experience they have is as follows:
* A user right clicks into a paragraph and selects "New comment"
* Our add-in detects the user is in a new paragraph and fetches the OOXML for the paragraph at the current selection
* As a result of our `getOoxml()` call, the user's cursor gets moved out of the new comment box and into the main document, causing them to accidentally edit the main document

Would it be straightforward for Word _not_ to update the user's cursor when fetching OOXML? It's an undesirable side effect of calling a really widely used method.
